modules/sops/default.nix

```
@@ -438,6 +447,7 @@ in
         after = [ "systemd-sysusers.service" ];
         environment = cfg.environment;
         unitConfig.DefaultDependencies = "no";
+        path = cfg.age.plugins;
```

```suggestion
        path = lib.mkForce cfg.age.plugins;
```

this was causing errors in `nix flake check`, with
> [environment.](error: The option `systemd.services.sops-install-secrets.environment.PATH' has conflicting definition values)